include/coglu.h

```diff
@@ -46,4 +54,9 @@ void add_shader_program(char *vertex_shader_path,
     glAttachShader(*shader_program_ptr, vertex_shader);
     glAttachShader(*shader_program_ptr, fragment_shader);
     glLinkProgram(*shader_program_ptr);
+    glValidateProgram();
```

Comment:
When testing this fails for me:

```
/usr/local/include/coglu/coglu.h: In function ‘add_shader_program’:
/usr/local/include/coglu/coglu.h:57:5: error: too few arguments to function ‘__glewValidateProgram’
   57 |     glValidateProgram();
      |     ^~~~~~~~~~~~~~~~~
```

Reply:
I suppose it should be `glValidateProgram(*shader_program_ptr);`?

Reply:
You're right! I will fix this soon enough